src/dbus_api/consts.rs

```diff
@@ -21,6 +21,7 @@ pub const POOL_NAME_PROP: &str = "Name";
 pub const POOL_UUID_PROP: &str = "Uuid";
 pub const POOL_HAS_CACHE_PROP: &str = "HasCache";
 pub const POOL_ENCRYPTED_PROP: &str = "Encrypted";
+pub const POOL_STATE_PROP: &str = "State";
```

Comment:
Would like to call this something less generic than "State".

Reply:
Sounds good! I'll go with `AvailableActions` as we decided in stand up.